spec/mysql2/client_spec.rb

```diff
@@ -887,11 +887,11 @@ def run_gc
       end.not_to raise_error
     end
 
-    it "should require an open connection" do
+    it "should not require an open connection" do
```

Comment:
Should we change this to `allows to escape on a closed connection`?

Reply:
I feel like it's basically the same thing. It's running in the `#escape` context.